src/tss2-esys/esys_crypto_ossl.h

```diff
@@ -109,6 +109,28 @@ TSS2_RC iesys_cryptossl_sym_aes_decrypt(
     uint8_t *iv,
     void *userdata);
 
+#if HAVE_EVP_SM4_CFB && !defined(OPENSSL_NO_SM4)
```

Comment:
Do we really need the check with OPENSSL_NO_SM4, wont, the check to determine HAVE_EVP_SM4_CFB be undefined if OpenSSL is built without SM4?

Reply:
the same as SM3 check. @williamcroberts